Src/StartMenu/StartMenuDLL/MenuContainer.cpp

```diff
@@ -7939,8 +7973,11 @@ HWND CMenuContainer::ToggleStartMenu( int taskbarId, bool bKeyboard, bool bAllPr
 						g_StdOptions[i].options=MENU_ENABLED|MENU_EXPANDED;
 				}
 				break;
+			case MENU_LOCK:
+				g_StdOptions[i].options = bLock ? MENU_ENABLED | MENU_EXPANDED:0;
```

Comment:
Please add also spaces around `:`. So that it is consistent with the rest of line.